configsuite/types.py

```diff
@@ -187,6 +192,8 @@ def _is_datetime(x):
 Bool = BasicType("bool", _is_bool)
 Date = BasicType("date", _is_date)
 DateTime = BasicType("datetime", _is_datetime)
+Anytype = BasicType("wildcard", _is_anytype)
```

Comment:
Please make the name of the basic type and its name correspond (`Anytype` vs `wildcard`). Also, I think we should make these private at some point. Together with `Type` and `Callable`, but that is for another day ;)

Reply:
fixed